src/app/services/admin.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AdminService {
+  public isAdmin = false;
```

Comment:
don't use public keyword, there are no need in it

Reply:
https://angular.io/tutorial/toh-pt4#create-the-messageservice

dudes from Google don't use it